form/form_test.go

```diff
@@ -128,21 +128,30 @@ func BenchmarkAppendTo(b *testing.B) {
 func TestAppendTo(t *testing.T) {
 	var arrayVal = [3]string{"1", "2", "3"}
 
-	var boolVal = true
+	var boolValT = true
+	var boolValF = false
 
 	var float32Val float32 = 1.2345
+	var float32Val0 float32
```

Comment:
I love the `*0` convention. Thanks for the considerable number of new tests too!